lab11-amanda/test/note-router-test.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+require('dotenv').config({path: `${__dirname}/../.env`});
+const superagent = require('superagent');
+const expect = require('expect');
+const server = require('../lib/server.js');
+const API_URL = `http://localhost:${process.env.PORT}`;
+let tempNote;
+
+describe('testing note routes', () => {
+  before(server.start);
+  after(server.stop);
+
+  describe('test POST /api/notes',() => {
+    it('should respond with a note', () => {
+      superagent.post(`${API_URL}/api/notes`)
```

Comment:
You need to make sure to return the result of `superagent` here and in the rest of the tests. `superagent` returns a promise, and that is how mocha knows when your test is done. If you don't return it, mocha will have no way of knowing and you could get false positives.